lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/components/RuntimeVersionManager.scala

```diff
@@ -401,7 +401,17 @@ class RuntimeVersionManager(
     */
   private def isEngineVersionCompatibleWithThisInstaller(
     manifest: Manifest
-  ): Boolean = CurrentVersion.version >= manifest.minimumRequiredVersion
+  ): Boolean = {
+    if (CurrentVersion.isDevVersion) {
+      logger.warn(
+        "Ignoring the minimum required engine version check " +
+        s"[${manifest.minimumRequiredVersion}] for the development version " +
+        s"[${CurrentVersion.version}]."
+      )
+      return true
+    }
+    CurrentVersion.version >= manifest.minimumRequiredVersion
```

Comment:
Hmmm, while in most cases this won't make a difference, this will fire the warning even if the dev version is `>= minimumRequiredVersion`. In which case the check is not really ignored - it would work anyway.

Any particular reason for why this is done in this order and not the suggested one?

Reply:
I thought that we will treat the dev version differently in any case, to make logic slightly simpler.

Anyway, I replaced it with the original solution.